### ISSUE TITLE
Windows high dpi borderless fullscreen

### ISSUE THREAD

**Issue by bananaturtlesandwich** [not-fl3/miniquad#326]
on windows when the config is set to have both fullscreen and high_dpi as true
```rust
miniquad::conf::Conf {
            window_title: ":)".to_string(),
            sample_count: 8,
            fullscreen: true,
            high_dpi: true,
            ..Default::default()
        }
```
the window will start up like this
<img width="960" alt="image" src="https://user-images.githubusercontent.com/71292624/202563081-5833275e-8e1e-4a82-a066-eb73bf6187f6.png">

**Comment by bananaturtlesandwich** [comment]
found a workaround for now by using winres to set this manifest
```xml
<?xml version="1.0" encoding="UTF-8" standalone="yes"?>
<assembly manifestVersion="1.0" xmlns="urn:schemas-microsoft-com:asm.v1" xmlns:asmv3="urn:schemas-microsoft-com:asm.v3">
    <asmv3:application>
        <asmv3:windowsSettings>
            <dpiAware xmlns="http://schemas.microsoft.com/SMI/2005/WindowsSettings">true</dpiAware>
            <dpiAwareness xmlns="http://schemas.microsoft.com/SMI/2016/WindowsSettings">PerMonitorV2</dpiAwareness>
        </asmv3:windowsSettings>
    </asmv3:application>
</assembly>
```

**Comment by bananaturtlesandwich** [comment]
Fixed with my pr #327
